x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/build_signal.ts

```diff
@@ -70,7 +70,7 @@ export const removeInternalTagsFromRule = (
   } else {
     const ruleWithoutInternalTags: Partial<OutputRuleAlertRest> = {
       ...rule,
-      tags: rule.tags.filter(tag => !tag.startsWith(INTERNAL_IDENTIFIER)),
+      tags: rule.tags.filter((tag: string) => !tag.startsWith(INTERNAL_IDENTIFIER)),
```

Comment:
Why is this line changing? It had the correct type without explicitly adding a string? I would suggest changing this back as when we have correct types in most areas there isn't an odd ball any or unknown and we don't have to re-type things:

<img width="557" alt="Screen Shot 2020-03-10 at 9 36 33 AM" src="https://user-images.githubusercontent.com/1151048/76329834-b181dc80-62b2-11ea-9276-69cca2267b9b.png">